src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/ExecuteRequests/ExecuteStringRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts.ExecuteRequests
+{
+    /// <summary>
+    /// Parameters for executing a query directly
+    /// </summary>
+    public class ExecuteStringParams : ExecuteRequestParamsBase
```

Comment:
Not sure why, but GitHub isn't expanding the ServiceIntegrationTests file. I added new unit tests to test the "getsqltext" method, which is the only place where the ExecuteStringRequest and ExecuteDocumentRequest code paths differ. Then I added a note:
```
﻿// NOTE: In order to limit test duplication, we're running the ExecuteDocumentSelection
// version of execute query. The code paths are almost identical.
```
I *can* go an duplicate all the tests for the ExecuteStringRequest, but I'd hate to add a ton of unnecessary tests. Do you want me to go ahead and do that?

That being said, I don't have unit tests for the params classes themselves, but since they're just storage classes that we haven't unit tested before, it seems unnecessary.